src/manifold.jl

```diff
@@ -33,30 +20,34 @@ properties.
 
 ## Arguments
 
-  - `g`: The residual function for the manifold.
```

Comment:
Similar to how our other packages work, I am making `manifold` st the IIP/OOP needs to align with the ODE problem